rbx_reflection/src/database.rs

```diff
@@ -39,6 +39,26 @@ impl<'a> ReflectionDatabase<'a> {
             enums: HashMap::new(),
         }
     }
+
+    /// Returns a list of superclasses for the provided class name. This list
+    /// will start with the provided class and end with `Instance` if the class
+    /// exists.
+    pub fn superclasses(&self, class_name: &str) -> Option<Vec<Cow<'a, str>>> {
```

Comment:
This method might be more convenient to use if returns `Vec<&ClassDescriptor>` - that way, consumers can get more information than just the name without having to look up the descriptor themselves, and won't have to destructure if all they want to do is e.g. iterate over superclasses

Reply:
That... is a very fair point. I have no objections to changing it.